scheduler-server/src/main/java/gov/nasa/jpl/aerie/scheduler/server/services/MerlinService.java

```diff
@@ -1,8 +1,8 @@
 package gov.nasa.jpl.aerie.scheduler.server.services;
 
 import gov.nasa.jpl.aerie.merlin.protocol.types.Duration;
-import gov.nasa.jpl.aerie.merlin.server.exceptions.NoSuchPlanException;
-import gov.nasa.jpl.aerie.merlin.server.models.PlanId;
+import gov.nasa.jpl.aerie.scheduler.server.exceptions.NoSuchPlanException;
```

Comment:
Similar comment about the 'Merline' name

Reply:
Just be sure to resolve this comment too if you become satisfied with the discussion on the other one.